Sources/Action/InputSubject.swift

```diff
@@ -23,7 +23,10 @@ public class InputSubject<Element>: ObservableType, Cancelable, SubjectType, Obs
 
     /// Indicates whether the subject has been isDisposed.
     public var isDisposed: Bool {
-        return _isDisposed
+        _lock.lock()
```

Comment:
I think that implementation is better as you say.
However, I am worried about implementing it like this because it is written in a style tailored to the implementation of other lines and implementation of PublishSubject on RxSwift.